### PR TITLE
Show favorites as cards

### LINKED ISSUE #329: As a Renter, I want the favorites page to be styled so the favorited apartments are displayed as cards.
**User story**

As a Renter, I want the favorites page to be styled so the favorited apartments are displayed as cards.

**Acceptance criteria**

**Definition of Done**

### PR DESCRIPTION
**Pull request by alldne**
- Closes #329
- Revises the style of cards
- Also makes pagination centered

Screen Shot 2019-12-06 at 2 53 30 AM

### CODE REVIEW

File: location/models.py [hunk 1]
```diff
@@ -33,6 +33,27 @@ def url_encoded_full_address(self):
     def google_map_url(self):
         return f"https://www.google.com/maps/search/?api=1&query={self.url_encoded_full_address}"
 
+    @property
+    def representative_image(self):
+        return self.apartment_set.exclude(image=None).first().image
+
+    @property
+    def representative_image_or_placeholder(self):
```
Comment (KShih) [inline location/models.py]: Will it conflict with this function in #336 ?
image

Comment (alldne) [reply location/models.py]: Let's merge #336 first and let me handle conflict

Comment (KShih) [reply location/models.py]: Thank you! I'll let you know as soon as it is merged.

Comment (alldne) [reply location/models.py]: @KShih Updated!

**Pull request merged**